### PR TITLE
[Plugins] Improve PO Token Guide

### PR DESCRIPTION
**Pull request by coletdjnz**
- ~~Removed usage of youtube embed to fetch PO Token - reports that YouTube is rolling out SABR on it which makes it difficult to find the po token.~~
- Remove `default` client from logged in example, as could mislead users to not supply po token to all clients. It is highly recommended that a po token is passed to all clients when logged in to avoid getting account blocked.

### CODE REVIEW

File: Extractors.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This process goes through manually obtaining a PO Token generated on YouTube in
 
 Steps:
 
-1. Open a browser and go to any video on YouTube Music or YouTube Embedded (e.g. https://www.youtube.com/embed/aqz-KE-bpKQ). **Make sure you are not logged in to any account!**
+1. Open a browser and go to any video on [YouTube Music](https://music.youtube.com). **Make sure you are not logged in to any account!**
```
Comment (bashonly) [inline Extractors.md:19]: do we need "Open any video on YouTube Music" as the next step here like we have in the "logged in" instructions?

Comment (coletdjnz) [reply Extractors.md]: good catch, made it consistent

**Review by bashonly — APPROVED** [review]

File: Extractors.md [hunk 2]
```diff
@@ -16,12 +16,13 @@ This process goes through manually obtaining a PO Token generated on YouTube in
 
 Steps:
 
-1. Open a browser and go to any video on YouTube Music or YouTube Embedded (e.g. https://www.youtube.com/embed/aqz-KE-bpKQ). **Make sure you are not logged in to any account!**
-2. Open the developer console (F12), then go to the "Network" tab and filter by `v1/player`
-3. Click the video to play and a player request will appear in the network tab
-4. In the request payload JSON, find the PO Token at `serviceIntegrityDimensions.poToken` and save that value
-5. Export cookies from the browser
-6. Pass the PO Token to yt-dlp using `--extractor-args "youtube:player-client=web,default;po_token=web+PO_TOKEN_VALUE_HERE"` with cookies (`--cookies COOKIES_FILE`)
+1. Open [YouTube Music](https://music.youtube.com) in a browser.  **Make sure you are not logged in to any account!**
```
Comment (coletdjnz) [inline Extractors.md:19]:
```suggestion
1. Open [YouTube Music](https://music.youtube.com) or YouTube Embedded (e.g. https://www.youtube.com/embed/aqz-KE-bpKQ) in a browser.  **Make sure you are not logged in to any account!**
```

Comment (coletdjnz) [reply Extractors.md:19]: since I have yet to see SABR be used

**Pull request merged**